user_tools/docs/user-tools-databricks-azure.md

```diff
@@ -92,20 +92,19 @@ A typical workflow to successfully run the `qualification` command in local mode
    2. installs Databricks CLI and configures the profile and the credentials to make sure the
       access credentials are stored in the file `~/.databrickscfg` on Unix, Linux, or macOS,
       or in another file defined by environment variable `DATABRICKS_CONFIG_FILE`.
-   3. installs Azure CLI and configures the profile and the credentials to make sure the Azure CLI
-      commands can access the ABFS resources `LOGS_CONTAINER` and `OUT_CONTAINER`.
+   3. installs Azure CLI and configures the credentials to make sure the Azure CLI
+      commands can access the ABFS resources `LOGS_CONTAINER`.
```

Comment:
That was something overlooked from before.
I was confused whether it is environment variable specific to azure
If this is the first time we mention "`LOGS_CONTAINER`", then we need to explain what is it.
For example
```
can access the ABFS storage container (called `$CONTAINER` here)..bla..bla
```